starter-code/index.html

```diff
@@ -47,7 +47,8 @@
     &copy; 2017 Code Fellows | Static Blog | Happy Ipsum!
   </footer>
 <!-- // TODO: Include any/all needed script tags -->
-<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
-
+<script src="scripts/jquery.js"></script>
```

Comment:
It is not necessary to replace the CDN link with a local copy of jQuery. Just for future reference.